README.md

```diff
@@ -127,12 +127,17 @@ Alternatively, you can just execute:
 sh ./scripts/create-workspace.sh
 ```
 
-Accessing the URL http://localhost:8087 will lead you to the app in production mode.
+Add the following line to your /etc/hosts file
+
+```
+127.0.0.1 host.docker.internal
+```
+
+Accessing the URL http://host.docker.internal:8087 will lead you to the app in production mode.
```

Comment:
May be in development mode? Let's not confuse users.

Reply:
Thanks, updated